src/Makefile.am

```diff
@@ -131,6 +132,7 @@ GRIDCOIN_CORE_H = \
 	ui_interface.h \
 	uint256.h \
 	util/memory.h \
+	util/strencodings.h
```

Comment:
missed a backslash. that is why Travis is barking

Reply:
Derp...fixed, thanks!